src/services/AbstractDFG.jl

```diff
@@ -971,6 +972,14 @@ function buildSubgraph(::Type{G},
     return destDFG
 end
 
+# Perhaps IIF.InMemDFGType should perhaps be incorporated as a DFG value, but this has been much debated in the past and hence just left as breadcrum.
+buildSubgraph(dfg::AbstractDFG,
+              variableFactorLabels::Vector{Symbol},
+              distance::Int=1;
+              solvable::Int=0,
+              sessionId::String = "",
+              kwargs...) = buildSubgraph(typeof(LightDFG(params=getSolverParams(dfg))), dfg, variableFactorLabels, distance, solvable=solvable, sessionId=sessionId, kwargs...)
```

Comment:
Fix typeof(LightDFG(params=getSolverParams(dfg))) to not make an extra object that is not used